### PR TITLE
#105 issue resolved. Added files for windows executable

### PR DESCRIPTION
**Pull request by thakran14**
## Issue that this pull request solves

Closes: # 105

## Proposed changes

Brief description of what is fixed or changed

Added files for windows executable. The executable file is in the dist folder.

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: wplay_exe.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# File for feeding the pyinstaller to create a windows executable file
+import os
+os.system('cmd /k "python -m wplay -h"')
```
Comment (rpotter12) [inline wplay_exe.py:3]: Codacy Issue found: [Starting a process with a partial executable path](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5339570)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5339570&bid=17479046)

File: wplay_exe.py (hunk 1 above)
Comment (rpotter12) [inline wplay_exe.py:3]: Codacy Issue found: [Starting a process with a shell: Seems safe, but may be changed in the future, consider rewriting without shell](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5339570)

Comment (thakran14) [reply wplay_exe.py:3]: okay. I'll try out an another way

Comment (thakran14) [reply wplay_exe.py:3]: Even though it is a partial executable path, it seems to be working well even when the project and the executable files were in different drives. I’ll try other ways also.

**Comment by xandao6** [comment]
@thakran14 send a message to me in slack, please.

**Comment by priyansh19** [comment]
@thakran14 check your PR template formating 
Make changes in it also. The checkbox is denoted "[x]" and you have used "[ x]" which doesn't denote anything

**Comment by thakran14** [comment]
> @thakran14 check your PR template formating
> Make changes in it also. The checkbox is denoted "[x]" and you have used "[ x]" which doesn't denote anything

I'm sorry for that. I have corrected the formating

**Pull request closed without merging**